client/client.go

```diff
@@ -56,6 +56,8 @@ type GlobalConfigItem struct {
 type RPCClient interface {
 	// GetAllMembers gets the members Info from PD
 	GetAllMembers(ctx context.Context) ([]*pdpb.Member, error)
+	// GetLeader gets the current leader and etcd leader from PD
+	GetLeaders(ctx context.Context) (leader *pdpb.Member, etcdLeader *pdpb.Member, err error)
```

Comment:
If, in some places, we both need members and leaders, then we need to call two RPCs?

Reply:
yes. here we try to not break capatibility of old method.

Reply:
how about define one response struct for this if we needs the informations of the follower?